.buildkite/pipeline.yml

```diff
@@ -808,6 +808,15 @@ steps:
         agents:
           slurm_gpus: 1
 
+      - label: "Perf: GPU implicit baro wave wrt h_elem"
+        key: "gpu_implicit_barowave_wrt_h_elem"
+        command:
+          - "julia --project -e 'using CUDA; CUDA.versioninfo()'"
+          - "nsys profile --trace=nvtx,cuda --output=gpu_implicit_barowave/report julia --color=yes --project=examples perf/benchmark_dump.jl --job_id gpu_implicit_barowave_wrt_h_elem --z_elem 25 --h_elem 12 --initial_condition DryBaroclinicWave --t_end 1mins --dt 1secs --dt_save_to_sol Inf --dt_save_to_disk Inf --device CUDADevice"
+        artifact_paths: "gpu_implicit_barowave_wrt_h_elem/*"
```

Comment:
Looks like the artifact is being generated at `climaatmos-ci/gpu_implicit_barowave/report.nsys-rep` instead!

Reply:
Ah, right, let's cancel

Reply:
What's the issue here? (And do you know what should be changed?)

Reply:
We didn't change the output path: `--output=gpu_implicit_barowave/report` to the correct path

Reply:
Updated.